starter-code/javascript/main.js

```diff
@@ -1,60 +1,43 @@
 var chronometer = new Chronometer();
-var btnLeft     = document.getElementById('btnLeft');
-var btnRight    = document.getElementById('btnRight');
-var minDec      = document.getElementById('minDec');
-var minUni      = document.getElementById('minUni');
-var secDec      = document.getElementById('secDec');
-var secUni      = document.getElementById('secUni');
-var milDec      = document.getElementById('milDec');
-var milUni      = document.getElementById('milUni');
+var btnLeft = document.getElementById("btnLeft");
+var btnRight = document.getElementById("btnRight");
+var minDec = document.getElementById("minDec");
+var minUni = document.getElementById("minUni");
+var secDec = document.getElementById("secDec");
+var secUni = document.getElementById("secUni");
+var milDec = document.getElementById("milDec");
+var milUni = document.getElementById("milUni");
 
+function printTime() {}
 
-function printTime() {
+function printMinutes() {}
 
-}
+function printSeconds() {}
 
-function printMinutes() {
+function printMilliseconds() {}
 
-}
+function printSplit() {}
 
-function printSeconds() {
+function clearSplits() {}
 
-}
+function setStopBtn() {}
 
-function printMilliseconds() {
+function setSplitBtn() {}
 
-}
+function setStartBtn() {}
 
-function printSplit() {
-
-}
-
-function clearSplits() {
-
-}
-
-function setStopBtn() {
-
-}
-
-function setSplitBtn() {
-
-}
-
-function setStartBtn() {
-
-}
-
-function setResetBtn() {
-
-}
+function setResetBtn() {}
 
 // Start/Stop Button
-btnLeft.addEventListener('click', function () {
-
+btnLeft.addEventListener("click", function() {
+  if (getElementsByClassName(start)) {
+    startClick();
```

Comment:
This should be `chronometer.startClick()`. Remember that you are calling a method of the class Chronometer, and in line 1 we have created an instance and saved it in the variable chronometer